ci

```diff
@@ -303,4 +336,160 @@ ci:main() {
     fi
 }
 
+#asdf:isNotInstalled() {
+#    bash:command:doesNotExist asdf
+#}
+#
+#asdf:install:asdfItself() {
+#    if asdf:isNotInstalled; then
+#        brew:install asdf
+#    fi
+#}
+#
+#asdf:plugin-add() { local packageName=$1
+#    asdf:install:asdfItself
+#    asdf plugin-add ${packageName}
+#}
+
+minikube:install:latestVersion() {
+#    asdf:plugin-add minikube
+    if bash:command:doesNotExist minikube; then
+        brew:cask:install minikube
+    fi
+}
+
+project:rootDirectory() {
+    pwd
+}
+
+minikube:start() {
+    if minikube:isNotRunning; then
+        minikube:cli start --mount --mount-string="$(project:rootDirectory):/dev-environment" --memory 8192 --cpus 2
+    fi
+}
+
+minikube:stop() {
+    minikube:cli stop
+}
+
+minikube:delete() {
+    minikube:cli delete
+}
+
+minikube:restart() {
+    minikube:stop
+    minikube:start
+}
+
+minikube:cli() { local args=$@
+    minikube:install:latestVersion
+    minikube ${args}
+}
+
+minikube:isNotRunning() {
+    ! (minikube:cli status | grep "minikube: Running")
+}
+
+minikube:devEnvironment:deploy() {
+    kubectl:cli apply -f src/devEnvironment/devEnvironment.yml
+    helm:install:nginx:ingress:controller
+}
+
+kubectl:cli() { local args=$@
+    minikube:start
+    kubectl ${args}
+}
+
+docker:cli:forMinikube() { local args=$@
+    minikube:start
+    eval $(minikube docker-env)
+    docker ${args}
+}
+
+minikube:dashboard() {
+    minikube:cli dashboard
+}
+
+minikube:ssh() {
+    minikube:cli ssh
+}
+
+minikube:ip() {
+    minikube:cli ip
+}
+
+dev:browser:open:fromMinikube() {
+    open "http://$(minikube:ip)"
+}
+
+helm:upgradeOrInstall() { local args=$@
+    helm:cli upgrade --install ${args}
+}
+
+helm:install() { local args=$@
+    helm:cli install ${args}
+}
+
+helm:cli() { local args=$@
+    helm:install:helmItself
+    minikube:start
+    helm init
```

Comment:
We might need to make sure kubecli is set to the minikube context.

Reply:
Minikube start points it there

Reply:
Ah I see. The minikube:start sets the context correctly.